app/assets/stylesheets/_sprites.scss

```diff
@@ -354,6 +354,9 @@
 .icon-send:before {
   content: "\e95c";
 }
+.icon-error:before {
+  content: "\e917";
+}
```

Comment:
The `icon-error` class is already defined with that content. I presume you want a different icon (with filled background) but that shouldn't shadow the existing name.